src/VPolygon.jl

```diff
@@ -132,3 +132,62 @@ List containing a singleton for each vertex.
 function singleton_list(P::VPolygon{N})::Vector{Singleton{N}} where {N<:Real}
     return [Singleton(vi) for vi in P.vertices_list]
 end
+
+"""
+    ∈(x::AbstractVector{N}, P::VPolygon{N})::Bool where {N<:Real}
+
+Check whether a given point is contained in a polygon in vertex representation.
+
+### Input
+
+- `x` -- point/vector
+- `P` -- polygon in vertex representation
+
+### Output
+
+`true` iff ``x ∈ P``.
+
+### Algorithm
+
+This implementation exploits that the polygon's vertices are sorted in
+counter-clockwise fashion.
+Under this assumption we can just check if the vertex lies on the left of each
+edge, using the dot product.
+
+### Examples
+
+```jldoctest
+julia> P = VPolygon([[2.0, 3.0], [3.0, 1.0], [5.0, 1.0], [4.0, 5.0]];
+                    apply_convex_hull=false);
+
+julia> ∈([4.5, 3.1], P)
+false
+julia> ∈([4.5, 3.0], P)
+true
+julia> ∈([4.4, 3.4], P)  #  point lies on the edge -> floating point error
+false
+julia> P = VPolygon([[2//1, 3//1], [3//1, 1//1], [5//1, 1//1], [4//1, 5//1]];
+                     apply_convex_hull=false);
+
+julia> ∈([44//10, 34//10], P)  #  with rational numbers the answer is correct
+true
+```
+"""
+function ∈(x::AbstractVector{N}, P::VPolygon{N})::Bool where {N<:Real}
+    @assert length(x) == 2
+
+    @inline function side(x, u, v)
```

Comment:
we may use the existent `right_turn(x, u, v)`  here

Reply:
I do not immediately see the mapping and if these functions do the same.

Reply:
`side(x, u, v) = right_turn(u, x, v)`

Reply:
Should I put `right_turn` to `helper_functions`?